src/musubi_tuner/wan_generate_video.py

```diff
@@ -1841,7 +1841,7 @@ def generate(
     seed_g.manual_seed(seed)
 
     # run sampling
-    latent = run_sampling(model, noise, scheduler, timesteps, args, inputs, device, seed_g, accelerator, is_i2v, arg_nocond=arg_nocond)
+    latent = run_sampling(models, noise, scheduler, timesteps, args, gen_settings, inputs, device, seed_g, accelerator, is_i2v, arg_nocond=arg_nocond)
```

Comment:
Not sure how that one happened (same up on 1834, assigning models) :(

I thought that I had taken upstream on this entire file and reintroduced your changes.

Sorry about the headache.